hack/provision.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# DESCRIPTION:
+# This is a script that bootstraps basic content in the OpenShift server
+# to facilitate development flows and users that want to kick the tires.
+# 
+# PROVISIONS:
+# * Users
+# ** test-cluster-admin
+# ** test-admin
+# * Docker registry
+# * OpenShift router
+# * OpenShift templates
+
+# INSTRUCTIONS:
+# $ git clone ...
+# $ make all
+# $ sudo <build output location>/openshift start
+# $ hack/provision.sh
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Export necessary environment variables for resources created on openshift start
+export ADMIN_KUBECONFIG=`pwd`/openshift.local.config/master/admin.kubeconfig
+export REGISTRY_KUBECONFIG=`pwd`/openshift.local.config/master/openshift-registry.kubeconfig
+export ROUTER_KUBECONFIG=`pwd`/openshift.local.config/master/openshift-router.kubeconfig
+export CA=`pwd`/openshift.local.config/master
```

Comment:
it's in $CWD... we decided not to allow configuring both the location of the "local config" and actual config files